cmd/agent/api/server.go

```diff
@@ -131,6 +130,9 @@ func StartServer(configService *remoteconfig.Service) error {
 	// such as streaming logs, which reset the deadlines back to zero)
 	handler := timeoutHandlerFunc(mux)
 
+	// Use a stack depth of 4 on top of the default one to get a relevant filename in the stdlib
+	logWriter, _ := config.NewLogWriter(5, seelog.ErrorLvl)
```

Comment:
in the comment you said `4` but here it is `5`. Could you please update the comment?